WBooks/Tabs/Library/LibraryController.swift

```diff
@@ -38,19 +38,19 @@ class MainMenuController: BaseViewController {
     
     private func configureNavigationBar() {
         navigationItem.title = "MAIN_MENU_NAV_BAR_TITLE".localized()
-        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(named: "ic_notifications"), style: UIBarButtonItem.Style.plain, target: self, action: nil)
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "ic_search"), style: UIBarButtonItem.Style.plain, target: self, action: nil)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage.notifications, style: UIBarButtonItem.Style.plain, target: self, action: nil)
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage.search, style: UIBarButtonItem.Style.plain, target: self, action: nil)
     }
 }
 
 // MARK: - UITableViewDelegate
-extension MainMenuController: UITableViewDelegate {
+extension LibraryController: UITableViewDelegate {
     func tableView(_ booksTable: UITableView, numberOfRowsInSection section: Int) -> Int {
         booksArray.count
     }
 
     func tableView(_ booksTable: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
```

Comment:
You can add "// MARK: - Private" before private methods to maintain an order, it will be easier to find when the class has a thousand lines xD